platform/spire/src/configuration.py

```diff
@@ -265,8 +265,8 @@ def get_kube_spec_vars(extra_kvs: dict=None) -> dict:
     return kvs
 
 
-def get_single_kube_spec(name: str, extra_kvs: dict=None) -> str:
-    templ = resource.get_resource("clustered/%s" % name).decode()
+def get_single_kube_spec(path: str, extra_kvs: dict=None) -> str:
+    templ = resource.package_resource(path).decode()
```

Comment:
I really don't like how all uses of these files now need to know whether to use `spire.resources.resource` or `resource.package_resource`; and the latter has bazel's weird `//` thing going on.  Is there any way we can improve this?

Reply:
With py_binary, we no longer get to remap the paths at which our dependencies get stuck into the output zipfile. (See https://github.com/sipb/homeworld/pull/487#discussion_r379573921.) This means that we need to pull in our resources from locations based on where they were in the original source tree, instead of being able to stick them all into a single unified `resources` directory and pull them all from there.

`spire.resources.resource("x")` pulls in a file `x` that was stored in `//spire/resources`. I could have also done this for (most of) the others -- we could `import dnsmasq` and do `dnsmasq.resource("kubernetes.yaml")` instead of `resource.package_resource("//dnsmasq:kubernetes.yaml")`, but that seemed like it would be potentially more confusing.

We can't do that for the paths with dashes in them -- like `dns-monitor` -- because those aren't valid python package names. There's probably a workaround for that, too, either based in renaming those directories in our source tree or renaming them in another way, but that also seemed messy.

I chose to reuse the `//package:filename` syntax; we didn't have to do that. It could be `resource.package_resource("dnsmasq", "kubernetes.yaml")` instead, but this way it seemed like it would be very clear that it was referencing a bazel package, and you'd know exactly where to look for the kubernetes.yaml, instead of having to figure out that bazel packages were being mapped to python packages before you could understand the reference.

I agree that this is somewhat ugly; it is not clear to me that there are easy improvements to the situation, but I'm interested in your input.

Maybe it works better to switch _everything_ to bazel's `//` syntax, for consistency?

Reply:
I think we should switch everything to a single syntax for consistency; I don't have an opinion on whether bazel's or two arguments is better.  I also think `resource.package_resource` is too long and it should just be `resource.get` or something

Reply:
Try that.